spark-etl/src/main/scala/geotrellis/spark/etl/config/EtlConf.scala

```diff
@@ -0,0 +1,68 @@
+package geotrellis.spark.etl.config
+
+import geotrellis.spark.etl.EtlJob
+import geotrellis.spark.etl.config.json._
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import org.apache.spark.SparkContext
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+
+class EtlConf(val credentials: Credentials, val datasets: List[Input], val output: Output) {
```

Comment:
I think its a cool idea to be able to have multiple inputs into an ETL job but `EtlJob` only takes one input, so it doesn't seem like having a list of them here is used?